include/tvm/meta_schedule/schedule_rule.h

```diff
@@ -180,7 +180,7 @@ class ScheduleRule : public runtime::ObjectRef {
    * \return The schedule rule created
    */
   TVM_DLL static ScheduleRule MultiLevelTilingWithIntrin(
-      String intrin_name, String structure, Optional<Array<String>> tile_binds,
+      Array<String> intrin_name, String structure, Optional<Array<String>> tile_binds,
```

Comment:
I understand why it is done (String -> Array<String>), but it should be rethink one more time due to API changing affects other places not only your own task

Reply:
Yes, new API changes were reverted to the original ones, while keeping the same new functionality.